packages/main/src/Select.js

```diff
@@ -203,6 +203,17 @@ class Select extends UI5Element {
 		return popover && popover.opened;
 	}
 
+	/**
+	 * Currently selected option
+	 * @readonly
+	 * @type { ui5-option }
```

Comment:
Perhaps 
`@returns` ui5-option and without `@multiple`

Reply:
Returns is usually for methods, not for getters - getters are treated as readonly properties (which have a type which is set or get - Boolean, String, ui5-option). I will remove the multiple.